woocommerce/woocommerce-ios
# [Mobile Payments] Two Gateways Enter, One Gateway Leaves

Closes: #6010

### Description
- Implement richer dialogs with user when two conflicting payment gateways are active
- Different dialogs are presented for administrators and shop managers
- Administrators are given a button to allow them to manage plugins

### Testing instructions
- Activate both the WCPay and Stripe plugins on a store
- Launch the app as a shop manager for that store
- Enter Menu > Settings (Gear) > In-Person Payments
- Ensure you are prompted to ask an administrator to deactivate one of the plugins
- Launch the app as an administrator for that store
- Enter Menu > Settings (Gear) > In-Person Payments
- Ensure you are prompted to deactivate one of the plugins and that the button takes you to wp-admin
- Deactivate one there
- On return, ensure the view refreshes and you can proceed to managing your card reader

### Screenshots

Admin:

<img src="https://user-images.githubusercontent.com/1595739/152618770-a05be56b-8eaa-4fd9-96dd-c74c34eec515.PNG" width=50% />

Shop Manager:

<img src="https://user-images.githubusercontent.com/1595739/152618794-26a544b1-1fd1-4985-b143-e55a24acc204.PNG" width=50% />

---
